lib/ansible/galaxy/collection/__init__.py

```diff
@@ -741,7 +741,7 @@ def install_collections(
         for fqcn, concrete_coll_pin in dependency_map.items():
             if concrete_coll_pin.is_virtual:
                 display.vvvv(
-                    "'{coll!s}' is virtual, skipping.".
+                    "'{coll!s}' is {concrete_coll_pin.type}, skipping.".
```

Comment:
Per above comments, this should probably be something like
```suggestion
                    "Encountered {coll!s}, skipping.".
```